package/graph.py

```diff
@@ -1,52 +1,41 @@
 import os
 import subprocess
 import sys
-from datetime import datetime
+import shutil
 
-def display(snippet_name, password):
-    # Retrieve the current time in HHMM format
-    current_time = datetime.now().strftime("%H%M")
-    
-    # Check if the provided password matches the current time
-    if str(password) != current_time:
-        raise ValueError("syntax error")
-    
-    # Proceed to copy code to clipboard if the password matches
-    snippet_path = os.path.join(
-        os.path.dirname(__file__), "stash", f"{snippet_name}.py"
-    )
 
+def display(snippet_name):
     try:
-        with open(snippet_path, "r") as file:
-            source_code = file.read()
+        base_dir = os.path.dirname(__file__)
+        snippet_path = os.path.join(base_dir, "stash", f"{snippet_name}.py")
```

Comment:
Remove this Line as its already defined, if you revert the deleted line changes.

Reply:
line 9 and 10 right ?

> Remove this Line as its already defined, if you revert the deleted line changes.

Reply:
line 10 only

Reply:
> line 10 only

ok

Reply:
done !! check once @jindalpriyanshu101

Reply:
Perfect!